docs/elasticsearchdataset-vct.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: zalando.org/v1
+kind: ElasticsearchDataSet
+metadata:
+  labels:
+    application: elasticsearch
+    role: data
+    group: simple
+  name: es-data-simple
+  namespace: es-operator-demo
+spec:
+  #replicas: 5
+  volumeClaimTemplates:
+  - metadata:
+      annotations:
+        volume.beta.kubernetes.io/storage-class: px-es-data-sc
+      name: data
+    spec:
+      storageClassName: px-es-data-sc
```

Comment:
Could you please use the `storageClassName: standard` here? It would be more generic. And please drop the `storage-class` annotation as it is not needed!